rules/coding-style/src/Rector/Variable/VariableVariablesRector.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\CodingStyle\Rector\Variable;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\PropertyFetch;
+use PhpParser\Node\Expr\Variable;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Core\RectorDefinition\CodeSample;
+use Rector\Core\RectorDefinition\RectorDefinition;
+use Rector\NodeTypeResolver\Node\AttributeKey;
+
+/**
+ * @see \Rector\CodingStyle\Tests\Rector\Variable\VariableVariablesRector\VariableVariablesRectorTest
+ * @see https://www.php.net/manual/en/language.variables.variable.php
+ */
+final class VariableVariablesRector extends AbstractRector
+{
+    public function getDefinition(): RectorDefinition
+    {
+        return new RectorDefinition('Ensure variable variables are wrapped in curly braces', [
+            new CodeSample(
+                <<<'PHP'
+function f()
+{
+    global $$foo->bar;
+}
```

Comment:
Better use longer funciton name than `f`, could be reserved in the future, like `fn()` is now.

We use `run()` as convention here

Reply:
Changed - sorry, it came from the PHP docs!